vizro-core/docs/pages/tutorials/first-dashboard.md

```diff
@@ -2,9 +2,9 @@
 
 There is no setup needed for your first dashboard, thanks to the amazing [Py.Cafe](https://py.cafe/).
 
-Follow the [Edit live on Py.Cafe](https://py.cafe/stichbury/vizro-iris-analysis) link below the dashboard to see the code and experiment with it.
+Follow the [Edit live on Py.Cafe](https://py.cafe/vizro./vizro-iris-analysis-0) link and you can see the code of the below dashboard and experiment with it.
```

Comment:
Was the user name not easy to fix?😬 `vizro.` looks a bit off as an URL? 

I liked your idea of vizro-team, vizro-official or vizro.team, vizro.official

Reply:
So far there was no fix although they are working on it. I would like to merge as is, then do another round whenever the fix becomes available. Otherwise this drags out too long..

Reply:
Done now